### PR TITLE
LPD-4319 Customize invitation email sender

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/LPD-4319

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -8902,6 +8902,10 @@ invisible-help=Check this to make the widget invisible to regular users.
 invisible-help[search]=Check this to make the widget invisible to regular users. The filter value from the configuration is applied by default, but users can still filter with other values via a URL parameter.
 invitation-email-body=Invitation Email Body
 invitation-email-body-description=Provide a localized template for the body of user invitation emails.
+invitation-email-sender-email-address=Invitation Email Sender Email Address
+invitation-email-sender-email-address-description=Define a specific email address for the sender of user invitation emails. If empty the inviter's email address is used.
```
Comment (andrea-ale-sbarra) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:8906]: @JamesAGarcia can you please review the translations?
Thank you!

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Code looks good, I've asked James to review the translations, please wait for his review before forwarding this PR.
Thank you!

**Comment by smottal** [comment]
ci:test:um

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**